pom.xml

```diff
@@ -23,7 +23,7 @@
     <jira-rest-client.version>5.2.2</jira-rest-client.version>
     <fugue.version>3.0.0</fugue.version>
     <!-- jenkins -->
-    <jenkins.version>2.277.4</jenkins.version>
+    <jenkins.version>2.320</jenkins.version>
```

Comment:
the[ bom atrifactID](https://github.com/jenkinsci/jira-plugin/pull/404/files#diff-9c5fb3d1b7e3b0f54bc5c4182965c4fe1f9023d449017cece3005d3f90e8e4d8R321-R322) should be updated too.

as the plugin was not using mask-classes or plugin first classloader before and there are no reports of breakage in ht e current core version, the core version could probably remain the same and an exclusion added wherever needed (if needed) to shutup the enforcer upper-bounds.

Reply:
it's a draft PR my idea was to change to a LTS.

Reply:
@olamy seems 2.332.1 LTS is out now, can we move forward with it?